internal/eval/deque_test.go

```diff
@@ -85,6 +87,87 @@ func dequeLPushIntStrMany(howmany int, deq eval.DequeI) {
 	}
 }
 
+func dequeLInsertIntStrMany(howMany int, beforeAfter string, deq eval.DequeI) {
+	const pivot string = "10"
+	const element string = "50"
+	deq.LPush(pivot)
+	for i := 0; i < howMany; i++ {
+		deq.LInsert(pivot, element, beforeAfter)
+	}
+}
+
+func BenchmarkBasicDequeLInsertBefore20(b *testing.B) {
+	for n := 0; n < b.N; n++ {
+		dequeLInsertIntStrMany(20, "before", eval.NewBasicDeque())
+	}
+}
+
+func BenchmarkBasicDequeLInsertBefore200(b *testing.B) {
+	for n := 0; n < b.N; n++ {
+		dequeLInsertIntStrMany(200, "before", eval.NewBasicDeque())
+	}
+}
+
+func BenchmarkBasicDequeLInsertBefore2000(b *testing.B) {
+	for n := 0; n < b.N; n++ {
+		dequeLInsertIntStrMany(2000, "before", eval.NewBasicDeque())
+	}
+}
```

Comment:
instead of so many cases, just check for a single case with a high number of elements - such as 20k. Same for others.

Reply:
Done.